pkgs/apps/wannier90/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, gfortran, blas, lapack, python2, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "wannier90";
+  version = "3.1.0";
+
+  nativeBuildInputs = [ gfortran ];
+  buildInputs = [
+    blas
+    lapack
+  ];
+
+  src = fetchFromGitHub {
+    owner = "wannier-developers";
+    repo = "wannier90";
+    rev = "v${version}";
+    hash = "sha256-+Mq7lM6WuwAnK/2FlDz9gNRIg2sRazQRezb3BfD0veY=";
+  };
+
+  postPatch = ''
+    rm -r test-suite/tests/testpostw90_{fe_kpathcurv,fe_kslicecurv,si_geninterp,si_geninterp_wsdistance}
+    patchShebangs test-suite/run_tests test-suite/testcode/bin/testcode.py
+  '';
+
+  configurePhase = ''
+    cp config/make.inc.gfort make.inc
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -p wannier90.x postw90.x $out/bin
+  '';
+
+  doCheck = true;
+  checkInputs = [ python2 ];
+  checkTarget = [ "test-serial" ];
+
+
+  hardeningDisable = [ "format" ];
+
+  meta = with lib; {
+    description = "Wannierization program, currently provided only serial (as parallel tests fail)";
```

Comment:
```suggestion
    Calculation of maximally localised Wannier functions
```

Reply:
Ah sorry, of course `description = ` and the quotations marks have to be kept. :)